docs/root/intro/version_history.rst

```diff
@@ -24,8 +24,10 @@ Version history
 * cli: added --config-yaml flag to the Envoy binary. When set its value is interpreted as a yaml
   representation of the bootstrap config and overrides --config-path.
 * cluster: Add :ref:`option <envoy_api_field_Cluster.close_connections_on_host_health_failure>`
+  to close tcp_proxy upstream connections when health checks fail.
```

Comment:
Looks like I accidentally deleted this in my previous PR. Opened #3322 to add this back in its own PR.